tests/contracts/fungible-token/output_4.json

```diff
@@ -1,5 +1,5 @@
 {
-  "gas_remaining": "1724",
+  "gas_remaining": "1720",
```

Comment:
It seems like the gas cost has increased in many of the tests because of this change. Is that because of some sort of overhead that dominates things for small maps?